stripe-buffer.el

```diff
@@ -136,9 +174,22 @@ Used by `stripe-table-mode' Only the first matching group will be painted."
                                     (goto-char end)
                                     (point))))))
                       ( overlay (apply get-overlay-create-function stripe-region)))
-                 (overlay-put overlay 'face stripe-highlight-face)
-                 (overlay-put overlay 'is-stripe t)
-                 (push overlay sb/overlays)))))
+
+                 (cond
+                  ((unless (and (fboundp 'dired-subtree--get-ov)
```

Comment:
Now that I see it again, this looks convoluted -- you have an extra unneeded conditional. You probably want `not` instead of `unless`. Or even better, remove the negation, and reverse the clauses.